tools/report-converter/codechecker_report_converter/report/output/html/html.py

```diff
@@ -436,7 +434,7 @@ def convert(
     file content change.
     """
     if not reports:
-        LOG.info(f'No report data in {file_path} file.')
+        LOG.info('No report data in %s file.', file_path)
```

Comment:
The major problem with using f-strings is that the string interpolation will happen **before** `LOG` is called. Which means that we will spend time crafting the log message even if the logger configuration would otherwise not print anything at all...

Reply:
Also, when you are "log-searching", i.e., trying to backtrack from an existing log message which piece of code was responsible for emitting that log, f-strings break established heuristics there, because you can no longer rollback a fully formatted log message to its "obvious" template placeholders.

Reply:
I found a major pull requests at #1886, #1911 which changed the previous, albeit non--f-string logging, where (similarly to f-strings!) the log message was **fully crafted** prior to calling of the actual logging handler, to using `printf`-style placeholders. (`f"X {foo}"` and `"X " + foo` and `"X %s".format(foo)` are all equivalent in terms of behav

Reply:
Thanks, this is the documentation I was looking for :) But again: `pylint` is also suggests using lazy string interpolation for the logging library.